### DISCUSSION TITLE
Cypress Integration

### DISCUSSION THREAD

**Discussion by joshmedeski** [nvim-neotest/neotest#296] [Ideas]
It would be really cool to add integration with [Cypress](https://www.cypress.io/). Although I don't know what would would need to be done or if it's even possible with the data Cypress provides to developers.

**Comment by codymikol** [comment]
It is possible, I'm currently writing a plugin for kotlin, but Cypress is also in my toolchain so I may tackle that next

**Reply by astrelinvl** [reply]
Is there any news on cypress?

**Reply by codymikol** [reply]
not at this time

**Comment by toto1384** [comment]
Any updates on this?
